lib/fetch/index.js

```diff
@@ -2021,7 +2021,9 @@ async function httpNetworkFetch (
       // into stream.
       const buffer = new Uint8Array(bytes)
       if (buffer.byteLength) {
-        fetchParams.controller.controller.enqueue(buffer)
+        try {
+          fetchParams.controller.controller.enqueue(buffer)
+        } catch {}
```

Comment:
@KhafraDev please review this. A wpt is failing without it: https://github.com/nodejs/undici/blob/main/test/wpt/tests/fetch/api/basic/stream-safe-creation.any.js

```
Uncaught exception in "D:\a\undici\undici\test\wpt\tests\fetch\api\basic\stream-safe-creation.any.js":
TypeError [ERR_INVALID_STATE]: Invalid state: ReadableStream is already closed
    at ReadableByteStreamController.enqueue (node:internal/webstreams/readablestream:1160:13)
    at fetchParams.controller.resume (D:\a\undici\undici\lib\fetch\index.js:2024:43)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
================================================================================================
Uncaught exception in "D:\a\undici\undici\test\wpt\tests\fetch\api\basic\stream-safe-creation.any.js":
TypeError [ERR_INVALID_STATE]: Invalid state: ReadableStream is already closed
    at ReadableByteStreamController.enqueue (node:internal/webstreams/readablestream:1160:13)
    at fetchParams.controller.resume (D:\a\undici\undici\lib\fetch\index.js:2024:43)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
================================================================================================
```